kyo-bench/src/main/scala/kyo/bench/BlockingBench.scala

```diff
@@ -0,0 +1,27 @@
+package kyo.bench
+
+import java.util.concurrent.locks.LockSupport
+
+class BlockingBench extends Bench.ForkOnly(()):
+
+    def block(): Unit =
+        LockSupport.parkNanos(100000) // 0.1 ms
```

Comment:
Can parkNanos be trivially interrupted? Perhaps we should have a blocking interruption benchmark, though it may not be very fair.

Reply:
Yes, Java's spec allows spurious wake ups but it shouldn't be a significant factor since that typically happens when the OS scheduler is under pressure, which isn't the case here since the workload is basically blocking threads.

Agree about introducing an interruption benchmark! I'll add it to my todo list.